ra-tls/ra-tls-server/lib/sgxsdk-ra-attester_u.c

```diff
@@ -1,18 +1,73 @@
 #include <assert.h>
 #include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
 
 #include <sgx_uae_service.h>
+#include <sgx_urts.h>
+#include <sgx_report.h>
+
+#ifdef RATLS_ECDSA
+#include <sgx_dcap_ql_wrapper.h>
+#include <sgx_default_quote_provider.h>
+#include <sgx_ql_lib_common.h>
+#include <sgx_error.h>
+#include <sgx_quote_3.h>
+#endif
 
 #include <ra.h>
 #include <ra-attester.h>
 #include <ias-ra.h>
 
-/* Untrusted code to do remote attestation with the SGX SDK. */
+#ifdef RATLS_ECDSA
+void ecdsa_get_quote
+(
+ sgx_report_t *report,
+ uint8_t* quote,
+ uint32_t* quote_len
+ )
+{
+	uint32_t quote_size = 0;
+
+	quote3_error_t qe3_ret = sgx_qe_get_quote_size(&quote_size);
+	if (SGX_QL_SUCCESS != qe3_ret) {
+		fprintf(stderr, "Error in sgx_qe_get_quote_size. 0x%04x\n", qe3_ret);
+		return;
+	}
+
+	// Get the Quote
```

Comment:
Plz delete this comment line. Basically the code itself already describe this fact.